backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtGL30Debug.java

```diff
@@ -0,0 +1,1629 @@
+/*******************************************************************************
+ * Copyright 2023 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.backends.gwt;
+
+import com.badlogic.gdx.utils.GdxRuntimeException;
+import com.google.gwt.webgl.client.WebGL2RenderingContext;
+
+import java.nio.Buffer;
+import java.nio.FloatBuffer;
+import java.nio.IntBuffer;
+import java.nio.LongBuffer;
+
+/** Does not extend from GwtGL20Debug as we would not have GL30 methods to override meaning we would need to implement all of GL30
+ * here as well.
+ *
+ * Instead, GL30 override methods are first, then GL20 override methods are beneath which were copied from GwtGl20Debug. */
```

Comment:
```suggestion
 * Instead, GL30 override methods are first, then GL20 override methods are beneath which were copied from GwtGL20Debug. */
```
Or maybe change this to a proper `{@link GwtGL20Debug}` (hope this was the right syntax)?

Reply:
Good catch, done.